modules/ocf_mirrors/files/rsyncd.conf

```diff
@@ -12,113 +12,142 @@ dont compress = *.gz *.tgz *.zip *.z *.rpm *.deb *.iso *.bz2 *.tbz
 [apache]
 	comment = apache project mirror
 	path = /opt/mirrors/ftp/apache
+	log file = /var/log/rsync/apache.log
```

Comment:
this file is template-able

Reply:
likely, maybe we could do some `concat` stuff here